.github/workflows/run-ode-regression.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: npm ci
       ## Run Cypress functional tests
       - name: Run tests
-        run: CYPRESS_baseUrl=${{ inputs.base_url }} CYPRESS_front_end_base_url=${{ inputs.base_url }} CYPRESS_ocpl_password='${{ secrets.ocpl_password }}' CYPRESS_admin_password='${{ secrets.admin_password }}' npm test
+        run: CYPRESS_baseUrl=${{ inputs.base_url }} CYPRESS_front_end_base_url=${{ inputs.base_url }} CYPRESS_ocpl_password='${{ secrets.ocpl_password }}' CYPRESS_admin_password='${{ secrets.admin_password }}' npm run cypress:acsf-parallel
```

Comment:
so this should actually be npm test-parallel ( add another script that is a copy of npm test but with parallelization)

Reply:
So would npm test-parallel test both the acsf and cms tests?